### DISCUSSION TITLE
Negative additional costs unreliable

### DISCUSSION THREAD

**Discussion by Floppe** [custom-components/nordpool#322] [Q&A]
Hi,

I started to use two Nordpool integrations. However, the issue I'm facing now is the second sensor that has negative margins which also is taken from another rest sensor.

`{{(states('sensor.electricity_sell_margin')|float) / 100}}`

where sensor.electricity_sell_margin is -0.24

Some hours per day are totally off. Like today.
![image](https://github.com/custom-components/nordpool/assets/238729/571e1805-e393-4e02-99fc-8bdf5875423b)

```raw_today: 
.... snip ....
- start: '2023-05-20T12:00:00+03:00'
  end: '2023-05-20T13:00:00+03:00'
  value: -0.202
- start: '2023-05-20T13:00:00+03:00'
  end: '2023-05-20T14:00:00+03:00'
  value: 0.225
- start: '2023-05-20T14:00:00+03:00'
  end: '2023-05-20T15:00:00+03:00'
  value: 0.185
- start: '2023-05-20T15:00:00+03:00'
  end: '2023-05-20T16:00:00+03:00'
  value: 0.231
- start: '2023-05-20T16:00:00+03:00'
  end: '2023-05-20T17:00:00+03:00'
  value: -0.24
- start: '2023-05-20T17:00:00+03:00'
  end: '2023-05-20T18:00:00+03:00'
  value: -0.108
``

Any thoughts?

**Comment by Hellowlol** [comment]
Most likely it’s related to https://github.com/custom-components/nordpool/pull/317 the pr should fix your issue
